delivery_driver/migrations/16.0.1.0.1/post-migration.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2024 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+import logging
+
+from odoo import SUPERUSER_ID, api
+
+_logger = logging.getLogger(__name__)
+
+
+def migrate(cr, version):
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        partners = env["carrier.driver"].search([("driver_id", "!=", False)])
```

Comment:
Don't you want partners?

```suggestion
        partners = env["carrier.driver"].search([("driver_id", "!=", False)]).mapped("driver_id")
```

Reply:
Ouch!! I updated this. Thanks!